app/services/idv/steps/document_capture_step.rb

```diff
@@ -31,14 +31,12 @@ def extra_view_variables
       private
 
       def native_camera_ab_testing_variables
-        ab_test = Idv::NativeCameraABTest.new
-        discriminator = flow_session[:document_capture_session_uuid]
-        skip_sdk = ab_test.bucket(discriminator) == :native_camera_only
+        bucket = ab_test.bucket(flow_session[:document_capture_session_uuid])
```

Comment:
good call on the spec. I also created `FakeAbTestBucket` in 60d8467ddeacbd4866329405c23adb03ed573af1 to simplify the test rather than use the haxx to reload, I think this is the pattern we should aim for when we can